cdap-docs/user-guide/source/tutorials/fitbit.rst

```diff
@@ -0,0 +1,238 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2017 Cask Data, Inc.
+    :description: The CDAP User Guide: Getting Started
+
+:hide-toc: true
+
+============================================== 
+Example: Analyzing and Masking IoT Device Data
+==============================================
+
+Introduction
+------------
+This tutorial demonstrates how to use CDAP's Data Preparation and Data Pipelines to clean, prepare, mask, and store IoT device data sent in JSON format.
+
+Scenario
+---------
+You receive FitBit device data in JSON format. You are interested in sharing the data with an outside contracter, but you need to mask the data to remove personally identify information before sharing (i.e., data masking). You will parse FitBit JSON data, extract the UNIX timestamp, masking the devices' IDs, hash the results, and store into a CDAP table. 
```

Comment:
`mask the devices' IDs`?  Or may be have it as simple past? Here we refer to `You will ...` but in the next line we say `You need to parse...`.